libs/application/templates/health-insurance/src/forms/HealthInsuranceForm.ts

```diff
@@ -43,7 +43,7 @@ export const HealthInsuranceForm: Form = buildForm({
         buildExternalDataProvider({
           title: m.externalDataTitle,
           id: 'approveExternalData',
-          subTitle: m.externalDataSubtitle,
+          subTitle: ' ',
```

Comment:
Does it work with only `''` instead of `' '`

Reply:
No if i use `''` it will use the default text instead.